Rspec/spec/rental_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../../rental'
+require_relative '../../teacher'
+require_relative '../../book'
+require 'rspec/autorun'
+
+describe Book do
```

Comment:
- [ ] As required for the project, kindly make sure you add tests to all classes that you created while implementing the Decorator design pattern and there should be one file per class. Right now, I noticed that there is still missing some files for some classes like `student_spec`, `decorator_spec`,...
**I suggest you check all expected classes, create new files  for them and add their tests(also apply on their methods)**. For some reminds about the Unit testing best practices lesson, you can check this **[article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/rspec_unit_test.md#unit-testing)**

Reply:
@tresorsawasawa 
Thank you for your useful feedback.
I have added the missing class which is decorator,
⚠️**Note:** we are to add tests for all classes only, not methods. 

Thank you.

Reply:
Hi @barakadanny 

I think you have to add also for all methods as mentioned in the **[project requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_unit_tests.md)**.
![Screenshot from 2022-11-23 20-44-15](https://user-images.githubusercontent.com/83042742/203624633-8bf92827-c9d6-48b4-a08c-dd125a037e33.png)

Reply:
Thank you. I am Working on it

Reply:
I have added to the decorator.